continuous_integration/environment-3.9.yaml

```diff
@@ -18,15 +18,13 @@ dependencies:
   - ipykernel
   - ipywidgets
   - jinja2
-  - joblib
```

Comment:
Last reference to this was removed in https://github.com/dask/distributed/pull/3040